### ISSUE TITLE
openfortivpn does not ask for password (v1.11.0 regression)

### ISSUE THREAD

**Issue by VojtechVitek** [adrienverge/openfortivpn#533]
After updating to v1.11.0, this command stopped working:

`sudo openfortivpn vpn.example.com:443 -u username -v`

```
DEBUG:  Setting cipher list to: HIGH:!aNULL:!kRSA:!PSK:!SRP:!MD5:!RC4DEBUG:  Gateway certificate validation failed.
DEBUG:  Gateway certificate digest found in white list.
INFO:   Connected to gateway.
ERROR:  Could not authenticate to gateway. Please check the password, client certificate, etc.
DEBUG:  HTTP status code 405
INFO:   Closed connection to gateway.
DEBUG:  server_addr: ******
DEBUG:  server_port: 443
DEBUG:  gateway_addr: ******
DEBUG:  gateway_port: 443
DEBUG:  Setting cipher list to: HIGH:!aNULL:!kRSA:!PSK:!SRP:!MD5:!RC4DEBUG:  Gateway certificate validation failed.
DEBUG:  Gateway certificate digest found in white list.
```

Before the update, I was being prompted for a password and then for a 2FA token.

I noticed this was reported at Red Hat bugzilla too: https://bugzilla.redhat.com/show_bug.cgi?id=1782877.

They suggest passing password via `-p` CLI argument (instead of stdin prompt). I confirm this indeed works fine. However, it's not safe to pass passwords this way, as it ends up in shell history etc.

**Comment by DimitriPapadopoulos** [comment]
What's in the config file?

**Comment by DimitriPapadopoulos** [comment]
See issue #507.

I suspect your config file defines an empty password. Starting with version 1.11 _openfortivpn_ accepts empty passwords.

As far as we know #508 fixes this.

**Comment by VojtechVitek** [comment]
@DimitriPapadopoulos you're right. My `/usr/local/Cellar/openfortivpn/1.11.0/etc/openfortivpn/config` file has an empty password. However, I didn't even know that I had this config file in the first place, as it was installed by brew automatically.

So this feels like a regression.. maybe in the Brew pkg?

**Comment by DimitriPapadopoulos** [comment]
The regressions is in _openfortivpn_ 1.11.0 and patch #508 has not yet been released in a formal version.

**Comment by DimitriPapadopoulos** [comment]
I'm closing this issue as a duplicate of #507, feel free to reopen if needed or add comments in either issues.

**Comment by mrbaseman** [comment]
the fix went into [release 1.12.0](https://github.com/adrienverge/openfortivpn/releases/tag/v1.12.0) but it was not working properly on Mac / BSD (see #554), so that patch is required in addition
